test/e2e/utils/utils.go

```diff
@@ -222,9 +222,12 @@ func TestWebhookServerTLS(f *framework.Framework, serviceName string) error {
 				{
 					Args: []string{
 						"--openssl=/usr/bin/openssl",
-						"-p",
+						"--mapping iana",
```

Comment:
Or alternatively (but I prefer the former):
```suggestion
						"--mapping",
						"iana",
```